src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -400,6 +424,28 @@ public override bool Execute()
         }
     }
 
+    private bool ProcessNettrace(string netTraceFile)
+    {
+        var OutputMibcPath = Path.Combine(OutputDir, Path.ChangeExtension(Path.GetFileName(netTraceFile), ".mibc"));
+        (int exitCode, string output) = Utils.TryRunProcess(Log,
+                                                            PgoBinaryPath!,
+                                                            $"create-mibc --trace {netTraceFile} --output {OutputMibcPath}",
+                                                            silent: true,
+                                                            debugMessageImportance: MessageImportance.Low);
+
+        if (exitCode != 0)
+        {
+            Log.LogError($"dotnet-pgo failed for {netTraceFile} {output}");
+            return false;
+        }
+
+        MibcProfilePath = MibcProfilePath!.Concat(new string[]{OutputMibcPath}).ToArray();
```

Comment:
This will do multiple allocations. Instead:

```suggestion
        Array.Resize(ref MibcProfilePath, MibcProfilePath.Length + 1);
        MibcProfilePath[^1] = OutputMibcPath;
```

Reply:
oops, yes! this is fine then - `MibcProfilePath = MibcProfilePath.Append(OutputMibcPath).ToArray();`

Reply:
I'm attempting to just change the property to be a `List<string>`

Reply:
Oh I didn't realize Append worked for Arrays in C# since it wasnt in the methods here https://docs.microsoft.com/en-us/dotnet/api/system.array?view=net-7.0 and the Explicit interface operator Add suggested it might return Not supported https://docs.microsoft.com/en-us/dotnet/api/system.array.system-collections-ilist-add?view=net-7.0